src/goImport.ts

```diff
@@ -6,12 +6,12 @@
 
 import vscode = require('vscode');
 import cp = require('child_process');
-import { getGoRuntimePath } from './goPath'
+import { getBinPath } from './goPath'
 import { parseFilePrelude } from './util'
 
 export function listPackages(): Thenable<string[]> {
 	return new Promise<string[]>((resolve, reject) => {
-		cp.execFile(getGoRuntimePath(), ["list", "all"], (err, stdout, stderr) => {
+		cp.execFile(getBinPath("gopkgs"), [], (err, stdout, stderr) => {
 			if (err && (<any>err).code == "ENOENT") {
 				vscode.window.showInformationMessage("The 'go' compiler is not available.  Install Go from http://golang.org/dl/.");
```

Comment:
Can you update the error message to refer to `gopkgs` being the binary that's missing?